core/src/main/resources/i18n/displayStrings.properties

```
@@ -560,9 +561,10 @@ portfolio.tab.history=History
 portfolio.tab.failed=Failed
 portfolio.tab.editOpenOffer=Edit offer
 
-portfolio.pending.invalidDelayedPayoutTx=Please do NOT send the Altcoin or Fiat payment but contact the Bisq \
-  developers at 'https://bisq.community' or the Keybase channel.\n\n\
-  {0}
+portfolio.pending.invalidDelayedPayoutTx=There is an issue with a missing or invalid transaction.\n\n\
+  Please do NOT send the Altcoin or Fiat payment but contact the Bisq \
+  developers at 'https://bisq.community' or the Keybase channel for further assistance.\n\n\
+  Error message: {0}
```

```suggestion
portfolio.pending.invalidDelayedPayoutTx=There is an issue with a missing or invalid transaction.\n\n\
  Please do NOT send the fiat or altcoin payment. Contact Bisq \
  developers on Keybase (https://keybase.io/team/bisq) or on the \
  forum (https://bisq.community) for further assistance.\n\n\
  Error message: {0}
```
Lowercased letters and added Keybase URL.